articles/tutorials/circuit-qft-tutorial.md

```diff
@@ -0,0 +1,298 @@
+﻿---
+title: Tutorial: Simulate a quantum circuit | Microsoft Docs
+description: Step-by-step tutorial to simulate a basic quantum circuit in Q#. Explore quantum gates and superposition.
+author: gillenhaalb
+ms.author: a-gibec@microsoft.com
+ms.topic: tutorial 
+ms.date: 10/06/2019
+uid: microsoft.quantum.circuit-tutorial
+# Customer intent: As a newcomer to the Q# (any amount of quantum experience), I want to know how to write and simulate a quantum circuit in Q# so that I can explore further on my own.
+---
+
+
+
+# Tutorial: Simulate quantum circuits in Q#
+Welcome to the Quantum Development Kit tutorial on setting up and simulating a basic quantum circuit. 
+It is important to note that the Q# language is far more than a new syntax for simulating quantum circuits. 
+If that were the only goal, a mere library of classical functions would have sufficed, because a quantum circuit is nothing more than the state's evolution through a specific sequence of gates---all of which can be fully expressed through matrix multiplication.
+Just as higher level classical languages like C and Python allow one to go beyond simulating sequences of AND and OR gates, Q# has been specifically designed with a computational paradigm shift in mind. 
+It is to be used for writing subroutines that, under the control of a classical host program and computer, execute on a separate quantum processing unit, thereby allowing developers to fully harness this classical-quantum symbiosis in full-scale quantum programs.
+
+However, if you are new to the ideas of quantum computation, or a researcher interested in fiddling with some ideas, small scale circuit simulation can nevertheless be a valuable tool---and, of course, it is still made easy with Q#.
```

Comment:
Great point. The researcher line was to avoid alienating anyone not "new to quantum computation," but rephrasing it like your suggestion solves the whole issue---I'll definitely fix it up, thanks